openrr-apps/src/bin/robot_teleop.rs

```diff
@@ -159,18 +159,23 @@ async fn main() -> Result<()> {
         }
         #[cfg(feature = "ros")]
         GamepadKind::Builtin(BuiltinGamepad::JoyGamepad) => {
+            arci_ros::init("joy_gamepad");
```

Comment:
@kaaatsu32329 @taiki-e 
Calling arci_ros::init() (= rosrust::init()) is not good idea.